lib/project/framework-project-base.ts

```diff
@@ -69,6 +71,25 @@ export class FrameworkProjectBase implements Project.IFrameworkProjectBase {
 		}
 	}
 
+	public getProperty(propertyName: string, configuration: string): any {
+		if(!this.projectInformation) {
+			this.$errors.fail("projectInformation object is empty");
+		}
```

Comment:
This looks more like an "assert" to me. Can this condition be triggered by user actions?